scripts/populate.js

```diff
@@ -3,7 +3,17 @@ var Chatty = (function(originalChatty){
 	var outputString = "";
 	var messagesArray = [];
 
-	
+	Chatty.getMessage = function () {
+		return messagesArray;
+	}
+
+	Chatty.newMessage = function () {
+		messagesArray.push(inputEl.value);
```

Comment:
This is definitely what we want the first action to be: add a string to our array. But we don't know that it will always be the value of the text input, for example, when the page loads we want to push the 5 strings from our JSON into the array. This is really good example of why parameters are useful, all we need to tell this function to do is what to do with the string, not where it comes from; we'll provide that information when we call the function. So taking into account the change to line 10 we would rewrite line 11 to something like messagesArray.push(msgString);